subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -444,34 +445,57 @@ static int cmd_wifi_disconnect(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int __wifi_scan_args_to_params(const struct shell *sh,
```

Comment:
No need to have underscores in this private function

Reply:
Fixed.